lib/state/appActions.js

```diff
@@ -62,6 +62,7 @@ appActions.errorStages = {
   STAGE_PICK_FILE : { code: 'E_SELECTING_FILE' , friendlyMessage: 'Error during file selection' },
   STAGE_READ_FILE : { code: 'E_READING_FILE' , friendlyMessage: 'Error trying to read file' },
   STAGE_UPLOAD : { code: 'E_UPLOADING' , friendlyMessage: 'Error uploading data' },
+  STAGE_METADATA_UPLOAD : { code: 'E_METADATA_UPLOAD' , friendlyMessage: 'Error uploading upload metadata'},
```

Comment:
Do you think this should be should be worded a little differently? Just thinking the rest of the messages make sense in general but this might be a little more cryptic